### DISCUSSION TITLE
Connect4 Steps

### DISCUSSION THREAD

**Discussion by Dobuses** [garrynewman/Control4.Jailbreak#8] [Q&A]
I want to thank you for this fantastic work around! Is there a link that provides instructions for programming an existing system to add new components such as door locks, lighting and garage door open / close control?

**Comment by cgbale** [comment]
> I want to thank you for this fantastic work around! Is there a link that provides instructions for programming an existing system to add new components such as door locks, lighting and garage door open / close control?

This is a good place to start: https://www.control4.com/docs/product/composer-he/user-guide/english/revision/P//composer-he-user-guide-rev-p.pdf
